### PR TITLE
nsis: update to 3.04

### PR DESCRIPTION
**Pull request by idleberg**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.x
Xcode 8.x

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by pmetzger** [comment]
@idleberg The build failed. You probably need to fix something up in order to make the update work.

**Comment by idleberg** [comment]
I'll have to look into it. There has been an [issue](https://sourceforge.net/p/nsis/bugs/1218/?limit=25#c964) with NSIS Menu in this release, maybe it's related.

**Comment by pmetzger** [comment]
@idleberg Any news?

**Comment by pmetzger** [comment]
@idleberg ?

File: devel/nsis/Portfile [hunk 1]
```diff
@@ -26,11 +26,11 @@ distfiles-append    nsis-${version}.zip
 extract.only-delete nsis-${version}.zip
 
 checksums           nsis-${version}-src.tar.bz2 \
-                    rmd160  b97f524fcda4dc333d73593c41130db5f8dddde1 \
-                    sha256  abae7f4488bc6de7a4dd760d5f0e7cd3aad7747d4d7cd85786697c8991695eaa \
+                    rmd160  9647acea3069fb8abb89853f3b394f4be7307728 \
+                    sha256  609536046c50f35cfd909dd7df2ab38f2e835d0da3c1048aa0d48c59c5a4f4f5 \
```
Comment (mf2k) [inline devel/nsis/Portfile]: Please add the `size` parameter here.

File: devel/nsis/Portfile [hunk 2]
```diff
@@ -26,11 +26,11 @@ distfiles-append    nsis-${version}.zip
...
                     nsis-${version}.zip \
-                    rmd160  806e501feff324f94e6d16e442cbaf6de4404837 \
-                    sha256  b53a79078f2c6abf21f11d9fe68807f35b228393eb17a0cd3873614190116ba7
+                    rmd160  795f9233fbbbd60876734a68397d172b1f47dec5 \
+                    sha256  22f3349fea453a45551745635c13e5efb7849ecbdce709daa2b2fa8e2ac55fc4
```
Comment (mf2k) [inline devel/nsis/Portfile]: Please add the `size` parameter here.

**Comment by idleberg** [comment]
> @idleberg Any news?

Upstream is aware of the problem, but there hasn't been a patch release yet

**Comment by idleberg** [comment]
Closed in favor of #6cdbb703cf585385ce3322768a6a633a99085e56

**Pull request closed without merging**